src/utils/Enums.ts

```diff
@@ -232,3 +232,15 @@ export enum StringKey {
   welcomeMessageLiveWelcomeStepContent = constants.welcomeMessageLiveWelcomeStepContent,
   welcomeMessageLiveWelcomeStepTitle = constants.welcomeMessageLiveWelcomeStepTitle,
 }
+
+export enum LaunchType {
+  cold = constants.cold,
+  /**
+   *  iOS only
+   */
+  unknown = constants.unknown,
```

Comment:
but unknown type does exist in iOS only and not in android so what are you suggesting?

Reply:
Based on our last discussion we agreed on porting the `unknown` type to Android as well since in some cases the launch type would be `null`, there doesn't have to be an actual enum value in the Android SDK though we can put an integer or string value (based on the enum type) in place of the original enum for `unknown` on Android.

Reply:
That was one of the options we discussed but we finally agreed on sending it as a string in the null check since we perform that in the implementation anyways, it was reviewed and retested after that fix.